src/ResourceManager/Sql/Commands.Sql.Test/ScenarioTests/ManagedInstanceKeyVaultKeyTests.ps1

```diff
@@ -0,0 +1,135 @@
+﻿# ----------------------------------------------------------------------------------
+#
+# Copyright Microsoft Corporation
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ----------------------------------------------------------------------------------
+		
+$mangedInstanceRg = "MlAndzic_RG"
+$managedInstanceName = "midemoinstancebc"
+$keyVaultName = "mitest-doNotDelete"
+$keyName = "mitest-key"
+$keyId = "https://mitest-donotdelete.vault.azure.net/keys/mitest-key/e97043455e14493ca05c3642549aedba"
+$keyVersion = "e97043455e14493ca05c3642549aedba"
+$tdeKeyName = $keyVaultName + "_" + $keyName + "_" + $keyVersion
+
+<#
+	.SYNOPSIS
+	Tests for managing TDE keyVaultKey in managed instance 
+#>
+function Test-ManagedInstanceKeyVaultKey
+{	
+	# Test Add
```

Comment:
Add Remove-AzureRmSqlManagedInstanceKeyVaultKey to one of these tests, along with a call to Get-AzureRmSqlManagedInstanceKeyVaultKey after the removal to ensure nothing is returned.

Reply:
We are unable to test Remove-AzureRmSqlManagedInstanceKeyVaultKey because of a production bug in management service, this bug is currently affecting multiple production APIs. I have included you in the email change with management service team, who are root-causing the issue

Reply:
Ah that's right.  We'll need to wait until that bug is fixed to merge this PR, and we can add a test for that before then.